docs/release/upgrading_to_v2_0.md

```diff
@@ -0,0 +1,45 @@
+# Upgrading to DAOS Version 2.0
+
+DAOS Version 2.0 is a major feature release, and it was a conscious
+design decision to **not** provide backwards compatibility with previous
+DAOS releases in favor of new feature development and stabilization.
+For future DAOS releases, a limited form of backward compatibility
+between two adjacent releases is planned.
+
+DAOS Version 2.0 introduces several protocol and API changes,
+as well as changes in the internal object layout,
+which make DAOS Version 2.0 incompatible with previous DAOS releases.
+It is therefore not possible to perform online or offline upgrades from
+previous DAOS releases to DAOS Version 2.0, while maintaining the user
+data in containers that have been created with previous DAOS releases.
+
+This means that an upgrade from DAOS Version 1.0 or 1.2 to
+DAOS Version 2.0 is essentially a new installation.
+Existing user data in DAOS should be backed up to non-DAOS storage
+before the upgrade, as the DAOS storage will need to be reformatted.
+For POSIX containers, there are two paths to perform such a backup:
+
+1. On a DAOS client where the DAOS POSIX container is dfuse-mounted,
+the _serial_ Linux `cp` command can be used to copy out the data.
```

Comment:
I believe this is not only the "cp" command which can benefit of the IL, so if yes/true, why not better say "any serial Linux copy command, like cp/..., can be used..." ?